src/groups/mqb/mqbi/mqbi_dispatcher.h

```diff
@@ -1048,6 +1054,7 @@ class DispatcherEvent : public DispatcherDispatcherEvent,
     messagePropertiesInfo() const BSLS_KEYWORD_OVERRIDE;
     bmqt::CompressionAlgorithmType::Enum
                         compressionAlgorithmType() const BSLS_KEYWORD_OVERRIDE;
+    bool                isOutOfOrderPush() const BSLS_KEYWORD_OVERRIDE;
```

Comment:
I know the semantics are obvious, but docs?

Reply:
These accessors seem to share the same doc
```
    // Return the value of the corresponding member.  Refer to the various
    // DispatcherEvent view interfaces for more specific information.
```

Reply:
As yes---I was seeing Doxygen docs where there are none (Doxygen won't copy docs of this way like BDE does).  That's fine.